docs/guides/honeypot.md

```diff
@@ -173,13 +174,19 @@ Now you are ready to start ContainerSSH:
 
 ```
 docker run -d \
-  --restart=always \
+  --restart=always --name containerssh \
   -v /srv/containerssh/:/etc/containerssh/ \
   -v /srv/containerssh/audit/:/var/log/containerssh/audit/ \
   --net=host \
   containerssh/containerssh:0.4.1
 ```
 
+Then we need to give container access to the audit log directory:
+
+```
+docker exec -d --user root containerssh chown 1022:1022 /var/log/containerssh/audit
```

Comment:
This gives container's user the write access to audit log directory

Reply:
Why not just call chmod on `/srv/containerssh/audit` outside the container?

Reply:
oh, I didn't know it's actually directly modifying the host folder!
updated!